include/deal.II/base/geometry_info.h

```diff
@@ -2932,8 +2941,6 @@ GeometryInfo<1>::is_inside_unit_cell (const Point<1> &p)
   return (p[0] >= 0.) && (p[0] <= 1.);
 }
 
-
-
```

Comment:
I think our coding style wants 3 lines between each function. Same in the next change.

Reply:
I'll just get back to the previous version here and at the other places.